src/Tests/dotnet-publish.Tests/GivenDotnetPublishPublishesProjects.cs

```diff
@@ -165,6 +166,7 @@ public void ItPublishesFrameworkDependentWithoutRid(string args)
         }
 
         [Theory]
+        [InlineData("-s --no-self-contained")]
```

Comment:
What about ``-s=true --no-self-contained``?

Also what about a shorthand for ``--no-self-contained`` as well?

Reply:
I thought `-s` stands for `--self-contained true`, if `-s=false` is needed, I'd like to add more cases.

Reply:
It probably is, but I seen test cases where it tests ``--self-contained=true``.

Reply:
Added, thanks

Reply:
I don't think we need a short-hand for `--no-self-contained`. The plan is for that to be the default in .NET 7.